### PR TITLE
example how to relay Ruuvitag data to MQTT

### PR DESCRIPTION
**Pull request by ikke-t**
Hi,

I have all my home-automation behind MQTT. It can then be further consumed by Node-Red or OpenHAB for example, which also store their states to influxdb for later analysis and visualization by Grafana. So I wanted to get my Sauna temp to MQTT, and while at it, I thought to contribute this as an example back to ruuvitag-sensor. Please review, and merge if it's good enough for you.

I tested it works on Fedora 29.

### CODE REVIEW

**Comment by ikke-t** [comment]
FYI, here's ansible playbook to enable this: https://github.com/RedHatNordicsSA/iot-hack/blob/master/run-ruuvi-to-mqtt.yml

File: examples/post_to_mqtt.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Get data from sensors and post it to specified mqtt channel in json-format
+
+Requires:
+    Ruuvitag sensor - pip3 install --user ruuvitag-sensor
+    Paho MQTT       - pip3 install --user paho-mqtt
+"""
```
Comment (ttu) [inline examples/post_to_mqtt.py]: You could add an example command here how to execute this file e.g. `post_to_mqtt.py --mac AA:BB:CC:DD:EE:FF --broker xxxx`

Comment (ikke-t) [reply examples/post_to_mqtt.py]: Thanks for the comments, it made it better. Didn't remember to use linter, good point. I added the example, along with link to Ansible automated version on how I handle sauna temperature at home. It set's the systemd file to keep the service going, along with dedicated user.

**Comment by ttu** [comment]
Thanks again for the example!

**Pull request merged**